src/components/TaskListView/TaskListView.css

```diff
@@ -0,0 +1,110 @@
+.done {
+    text-decoration: line-through;
```

Comment:
What do you think about having a done task's text having a text color with lower opacity?
Example: incomplete tasks in the table can stay the same style, but completed tasks could have the color of rgb(0,0,0,0.5)? it will reduce their opacity and further (visually) distinguish the incomplete tasks from the completed ones?

Reply:
![Screenshot 2023-07-22 at 1 42 32 AM](https://github.com/SethBurns/bare-minimum/assets/122247155/dfc7c8bc-0a5d-4439-ac04-c420cae98e87)
Would look something like this

Reply:
OOOOH LOVE THIS!!!